exporter/observiqexporter/config.go

```diff
@@ -37,12 +38,18 @@ type Config struct {
 	APIKey string `mapstructure:"api_key"`
 	// Secret key for authenticating with the ingestion endpoint (required if no APIKey)
 	SecretKey string `mapstructure:"secret_key"`
-	// Endpoint URL; Defines the ingestion endpoint (optional)
+	// Endpoint URL; Defines the ingestion endpoint
 	Endpoint string `mapstructure:"endpoint"`
-	// ID that identifies this agent (optional)
+	// ID that identifies this agent
 	AgentID string `mapstructure:"agent_id"`
-	// Name that identifies this agent (optional)
+	// Name that identifies this agent
 	AgentName string `mapstructure:"agent_name"`
+	// DialerTimeout is the amount of time to wait before aborting establishing the tcp connection when making
+	// an http request
+	DialerTimeout time.Duration `mapstructure:"dialer_timeout"`
```

Comment:
It looks like this could be added to `confignet` fairly easily. If you agree, would you mind opening a issue?

Reply:
Opened one here: https://github.com/open-telemetry/opentelemetry-collector/issues/4331